runtime/compiler/runtime/SymbolValidationManager.cpp

```diff
@@ -111,6 +115,26 @@ TR::SymbolValidationManager::SymbolValidationManager(TR::Region &region, TR_Reso
       _alreadyGeneratedRecords.insert(
          new (_region) ArrayClassFromComponentClassRecord(arrayClass, component));
       }
+
+#if defined(J9VM_OPT_JITSERVER)
+   if (TR::CompilationInfo::get()->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
```

Comment:
We can't use that method here, since SVM is created from `J9::Compilation` constructor, and `_isOutOfProcessCompilation` is not set yet. But if we move if outside of the constructor then it can be changed.